src/Phoenix/PhoenixClient.ts

```diff
@@ -161,12 +161,26 @@ export class PhoenixClient {
 
   public async isDbAcountWhitelisted(): Promise<boolean> {
     try {
+      TelemetryProcessor.traceStart(Action.PhoenixDBAccountAllowed, {
```

Comment:
This method returns a timestamp that you can pass down to traceFailure and traceSuccess to track the overall time.

Reply:
Got it